internals/webpack/webpack.base.babel.js

```diff
@@ -34,6 +34,21 @@ module.exports = (options) => ({
     }, {
       test: /\.jpe?g$|\.gif$|\.png$/i,
       loader: 'url-loader?limit=10000',
+    }, { 
+      test: /\.woff(\?v=\d+\.\d+\.\d+)?$/, 
+      loader: 'file?name=fonts/[name].[hash].[ext]&mimetype=application/font-woff' 
+    }, { 
+      test: /\.woff2(\?v=\d+\.\d+\.\d+)?$/, 
+      loader: 'file?name=fonts/[name].[hash].[ext]&mimetype=application/font-woff'
+    }, { 
+      test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, 
+      loader: 'file?name=fonts/[name].[hash].[ext]&mimetype=application/octet-stream' 
+    }, { 
+      test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, 
+      loader: 'file?name=fonts/[name].[hash].[ext]' 
+    }, { 
+      test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, 
+      loader: 'file?name=images/[name].[hash].[ext]&mimetype=image/svg+xml' 
```

Comment:
Instead of already using the file loader, add `.svg` the the `.jpg|.gif|.png` loader above so it also uses the `url-loader`!